### ISSUE TITLE
SlugRedirect objects should be created on editing a slug

### ISSUE THREAD

**Issue by mhl** [mysociety/pombola#1476]
In the Django admin interface, if you edit the slug field of an object whose views support slug redirection, a new SlugRedirect should be created.

**Comment by tmtmtmtm** [comment]
@mhl can you describe this in terms of the issue that it creates? I'm guessing this is about URLs stopping working when someone edits the slug of something?

**Comment by mhl** [comment]
Yes, exactly - if an admin changes the slug of a person, say, there's no redirection from the old URL. (I can't remember whether this was provoked by it actually happening or not, I'm afraid.)
